### ISSUE TITLE
hosted_zone/vpc_id is defined as number but vpc_id is always string

### ISSUE THREAD

**Issue by codecheetos** [aws-ia/terraform-aws-route53-recovery-controller#23]
I'm working on a spike with route53 ARC using the aws-ia terraform module. The examples are not very intuitive and good to have some docs for examples.

For specific question - in variables.tf, the hosted_zone/vpc_id is defined as number; but AFAIK its usually defined as string in console. [https://github.com/aws-ia/terraform-aws-route53-recovery-controller/blob/main/variables.tf#L87 ](https://github.com/aws-ia/terraform-aws-route53-recovery-controller/blob/main/variables.tf#L87)any ideas how to make this work?

My terraform environment/version does not allow me to perform experimental syntax and cannot use `optional` etc.

**Comment by drewmullen** [comment]
👋 thanks for opening this issue and for your feedback. It is certainly true that documentation can be improved and i would _very much welcome_ input from you as a user approaching this module!

regarding the vpc_id parameter, that definitely looks like a bug to me. we pass the `hosted_zone` contents (if they exist) to the [route53_zone data source](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/route53_zone#vpc_id)

I can get that PR pushed shortly to hopefully unblock you

**Comment by drewmullen** [comment]
`vpc_id` type error has been fixed in v1.0.2

Please share any feedback you have for examples docs in a separate issue

thank you for reporting!

**Comment by drewmullen** [comment]
@codecheetos 
I made a couple of small updates to the example readme as part of a long-coming refactor: https://github.com/aws-ia/terraform-aws-route53-recovery-controller/releases/tag/v1.0.3

LMK if you have further feedback
